src/bootstrap/step.rs

```diff
@@ -789,6 +786,10 @@ pub fn build_rules<'a>(build: &'a Build) -> Rules {
          .dep(move |s| s.name("tool-build-manifest").target(&build.config.build).stage(0))
          .run(move |_| dist::hash_and_sign(build));
 
+    rules.install("install", "path/to/nowhere")
+         .dep(|s| s.name("default:dist"))
```

Comment:
Oh yeah so we'll definitely still need to dist things like libstd/librustc, but we just don't need to dist the source tarball, right?